packages/xchain-litecoin/src/utils.ts

```diff
@@ -0,0 +1,347 @@
+import * as Litecoin from 'bitcoinjs-lib' // https://github.com/bitcoinjs/bitcoinjs-lib
+import * as sochain from './sochain-api'
+import { Address, Balance, Fees, Network, TxHash, TxParams } from '@xchainjs/xchain-client'
+import { AssetLTC, assetToString, BaseAmount, baseAmount, assetToBase, assetAmount } from '@xchainjs/xchain-util'
+import { LtcAddressUTXOs, LtcAddressUTXO } from './types/sochain-api-types'
+import { FeeRate, FeeRates, FeesWithRates } from './types/client-types'
+import { BroadcastTxParams, DerivePath, UTXO, UTXOs } from './types/common'
+import { MIN_TX_FEE } from './const'
+import coininfo from 'coininfo'
+
+const TX_EMPTY_SIZE = 4 + 1 + 1 + 4 //10
+const TX_INPUT_BASE = 32 + 4 + 1 + 4 // 41
+const TX_INPUT_PUBKEYHASH = 107
+const TX_OUTPUT_BASE = 8 + 1 //9
+const TX_OUTPUT_PUBKEYHASH = 25
+const DUST_THRESHOLD = 1000
+
+function inputBytes(input: UTXO): number {
+  return TX_INPUT_BASE + (input.witnessUtxo.script ? input.witnessUtxo.script.length : TX_INPUT_PUBKEYHASH)
+}
+
+/**
+ * Compile memo.
+ *
+ * @param {string} memo The memo to be compiled.
+ * @returns {Buffer} The compiled memo.
+ */
+export const compileMemo = (memo: string): Buffer => {
+  const data = Buffer.from(memo, 'utf8') // converts MEMO to buffer
+  return Litecoin.script.compile([Litecoin.opcodes.OP_RETURN, data]) // Compile OP_RETURN script
+}
+
+/**
+ * Get the vault transaction fee.
+ *
+ * @param {UTXOs} inputs The UTXOs.
+ * @param {Buffer} data The compiled memo.
+ * @param {FeeRate} feeRate The fee rate.
+ * @returns {number} The fee amount.
+ */
+export function getVaultFee(inputs: UTXOs, data: Buffer, feeRate: FeeRate): number {
```

Comment:
The language of "vault" and "normal" was deprecated when we moved from asgardex to xchainjs. XChainjs is agnostic and shouldn't have the concept of "vault".

The only difference between the two is the presence of the memo.

Request replace both functions. with a single "getFee()`, where the memo is optional